### ISSUE TITLE
[Bug] Black screen after surviving fall + strange soft-lock after diving/DBNO-ing

### ISSUE THREAD

**Issue by Poggas1256** [JonnyBro/beatrun#65]
(Black screen) The screen stays black after falling and not dying [Godmode/Buddha] (i was specifically falling with the DBNO dive, don't know if that affects the results in any way.
(Soft-lock) stuck in place after diving to a slightly raised surface in front of me and attempting to roll instead of continuing to slide

I am using the reanimated beatrun animations, not the default ones. I don't have any addons that would affect beatrun.

https://github.com/JonnyBro/beatrun/assets/143871014/11c6370a-6875-4e6e-8f2e-4c62e3e9ac03

**Comment by JonnyBro** [comment]
> I don't have any addons that would affect beatrun.

Are you sure? I saw on the video that you have addons beside beatrun

**Comment by Poggas1256** [comment]
yeah i have addons besides beatrun, but 99% of them are all weapons, viewbob, model/texture replacers, vmanip stuff, etc, etc.
the reason im not worried about viewbob is [this addon](https://steamcommunity.com/sharedfiles/filedetails/?id=2922515253) which adds a layered system to viewbob/calcview in gmod, making them not replace eachother, but stack on instead

the ONE addon i think even has a possibility of causing this is [this one](https://steamcommunity.com/sharedfiles/filedetails/?id=2512464000) which allows the player to walk and jump off of props correctly, instead of all glitchy-like. ill try to disable it and do the glitch again, and ill get back to you on the results (along with disabling all my mods besides beatrun if it doesnt fix it)

**Comment by Poggas1256** [comment]
ok so the soft-lock this stopped happening??? i didnt change my addons and went on and did it with the same addons as a test and it just didnt happen anymore???

The black screen after not dying to a fall still happens of course. this is a problem with beatrun, i remember it happening back when i had EA to the original beatrun (i even tried fixing it myself in the code just by getting rid of the black screen effect in the files, but i just dont know where its located)

im pretty sure the black screen after not dying to a fall is just because it doesnt check if you died from falling, it just checks that you WOULD have died, and that you hit the ground, so if you find the code hopefully its not too hard to fix

**Comment by dataesucsdeek** [comment]
what a nice reshade

**Comment by Mikeyisthatboi** [comment]
dude black screen after dying in godmode thats just part of the addon when you die from a height you can't survive that effect triggers just equip runner hands and press r.

**Comment by Mikeyisthatboi** [comment]
> dude black screen after dying that's just part of the addon when you die from a height you can't survive that effect triggers just equip runner hands and press r.
